tests/SourcemapToolkit.SourcemapParser.UnitTests/SourceMapUnitTests.cs

```diff
@@ -118,5 +119,230 @@ public void GetMappingEntryForGeneratedSourcePosition_NoExactMatchHasSimilarOnDi
             // Asset
             Assert.AreEqual(matchingMappingEntry, result);
         }
-    }
+
+		private MappingEntry getSimpleEntry(SourcePosition generatedSourcePosition, SourcePosition originalSourcePosition, string originalFileName)
+		{
+			return new MappingEntry
+			{
+				GeneratedSourcePosition = generatedSourcePosition,
+				OriginalSourcePosition = originalSourcePosition,
+				OriginalFileName = originalFileName
+			};
+		}
+
+		private SourcePosition generateSourcePosition(int lineNumber, int colNumber = 0)
+		{
+			return new SourcePosition
```

Comment:
Let's just inline these calls, its easier to read. Also, we should be explicit with the column numbers to make sure we're getting coverage there